src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,46 @@
+# How to run the tests
+This shows how to run the tests on a Raspberry Pi. On other platforms, things should be similar. 
+
+## Building on the desktop PC
+(to be extended)
+
+## Building directly on the Pi
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
+- Locally cloned repository
+
+Build System.Device.Gpio.dll:
+```
+cd src/System.Device.Gpio
+dotnet build System.Device.Gpio.sln
```

Comment:
I'm not 100% sure that this will always work, It might be that it is working now by chance, but the ideal workflow should be to just call /build.sh, or in the worst case, ./build.sh -restore which would install dotnet for you along with all of the dependencies that we require for building. Anyways, I'm fine with keeping this as is for now and just fix whatever issue you were hitting above when calling ./build.sh and once we do change these instructions.

Reply:
I always use dotnet build as well but I first do full build and force using dotnet from the `.dotnet` directory which got created

Reply:
yeah, that is supposed to work as well. But never calling build.sh, and just using a random dotnet cli most likely not work in the future, so I'd be happier if we could use the one that gets restored by our build instead.

Reply:
I think you get an error if the correct CLI version is not installed. I personally prefer having the CLI installed globally once instead of into each working copy. Maybe both methods should be described (once they work...)

Reply:
@pgrawehr, in order to use the CLI which got created by the build when you also have a global installation you should also set environment variable: `export DOTNET_MULTILEVEL_LOOKUP=0` (or `set ...` on Windows) to use only framework next to the dotnet.exe and not try to search for other frameworks